01-node-tutorial/answers/writeWithPromisesAwait.js

```diff
@@ -0,0 +1,26 @@
+const { writeFile, readFile } = require("fs").promises;
+
+async function writer() {
+  try {
+    await writeFile("temp.txt", "Line 1\nLine 2\nLine 3\n");
```

Comment:
Requires changes:
I believe the assignment wanted you to write to the file 3 times, instead of using `\n`. This would resemble the thenable example below, where you write to the same file 3 times as well.
```js
await writeFile("...", "...");
await writeFile("...", "...", { flag: "a" });
await writeFile("...", "...", { flag: "a" });
```